Flutter/pdf-viewer/select-deselect-annotations.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: Select and Deselect Annotations in Flutter PDF Viewer widget | Syncfusion
+description: Learn here all about select and deselect annotations in PDF documents using the Syncfusion Flutter PDF Viewer (SfPdfViewer) widget and more.
+platform: Flutter
+control: SfPdfViewer
+documentation: ug
+---
+
+# Select and Deselect Annotations in Flutter PDF Viewer widget (Syncfusion)
+
+This section will go through the various functions available in the `SfPdfViewer` for selecting and deselecting annotations in a PDF document.
+
+## Select an annotation
+
+You can select an annotation by simply tapping on the annotation using touch or mouse. When the annotation is selected, the selection border (selector) appears, indicating that the annotation is selected. 
+
+### Select an annotation programmatically
+
+You can select an annotation programmatically by providing the annotation instance as the parameter to the `selectAnnotation` method of PdfViewerController. The annotation instance can be found in the `getAnnotations` method of the PdfViewerController. The following example explains how to select the first annotation in the annotation collection.
```

Comment:
You can select an annotation programmatically by providing the annotation instance as the parameter to the `selectAnnotation` method of the PdfViewerController. The annotation instance can be found in the `getAnnotations` method of the PdfViewerController. The following example explains how to select the first annotation in the annotation collection.